### PR TITLE
Add integration tests for block_on_spring

### PR DESCRIPTION
**Pull request by gracetangg**
# Pull Request
### Description
Creates integration tests for block_on_spring app.

This pull request addresses #86 

- [x] make sure you are pulling from a dev branch on your fork
- [x] check for code linting

### CODE REVIEW

**Comment by arichar6** [comment]
Did you copy the test problem from the most recent version of the block-on-spring code? I think there are more docstrings in [the version here](https://github.com/NRL-Plasma-Physics-Division/block-on-spring).

Also, I would like the problem configuration for this test to be defined directly as a python dictionary, rather than by importing a toml file. That way if anyone looks at how the test is written, they will see a different approach to defining their problem. You can copy the code from the repo linked above.

**Comment by gracetangg** [comment]
@arichar6 should test for the leapfrog method too, or just the forward euler?

**Comment by arichar6** [comment]
Good question. How about test both. But you don't need to run very many time steps. Something like 10 steps should be plenty.

**Comment by arichar6** [comment]
Just in case I wasn't clear, I meant that you can test both, and that the tests for both of the solvers only need to run for 10 time steps each.

File: tests/fixtures/block_on_spring/block_on_spring.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
+"""Use turboPy to compute the motion of a block on a spring"""
+import numpy as np
+import pytest
+# import xarray as xr
+# import matplotlib.pyplot as plt
```
Comment (arichar6) [inline tests/fixtures/block_on_spring/block_on_spring.py:5]: You don't need these lines.
```suggestion
```

**Comment by gracetangg** [comment]
@arichar6 Thank you! I added a test for BackwardsEuler while I was at it.

File: tests/fixtures/block_on_spring/block_on_spring.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        print(data)
+
+    def csv_diagnose(self, data):
+        self.csv.append(data)
+
+
+class ForwardEuler(ComputeTool):
+    """Implementation of the forward Euler algorithm
+
+    y_{n+1} = y_n + h * f(t_n, y_n)
```
Comment (arichar6) [inline tests/fixtures/block_on_spring/block_on_spring.py:73]: Nice, the docstrings are here 😄

**Comment by arichar6** [comment]
Closes #86

**Pull request merged**